### DISCUSSION TITLE
Multiple birds in one recording

### DISCUSSION THREAD

**Discussion by matthew73210** [mcguirepr89/BirdNET-Pi#899] [Q&A]
Just a quick question, that might have already been answered. Git search didn't help.
How does the system react when there are multiple bird calls from different species in one recording? I can clearly make out a thrush and a common buzzard in the same recording but it always goes for the buzzard.

**Comment by lloydbayley** [comment]
That's a good question. From what I understand, it will take the one that was the best match from the audio. So, ultimately, the bird with the stronger call. I usually hope that the other bird calls again and gets pole position in the next detection.

**Comment by ehpersonal38** [comment]
https://github.com/mcguirepr89/BirdNET-Pi/discussions/547#discussioncomment-3672053

**Reply by matthew73210** [reply]
Cheers
